src/frontend/src/lib/components/wizards/auth/views/PickAuthenticationMethod.svelte

```diff
@@ -133,30 +130,15 @@
         </Button>
       </Tooltip>
     {/if}
-    {#if $AUTH_FLOW_UPDATES}
-      <div class="border-border-tertiary border-t"></div>
-      <div class="flex flex-row items-center justify-between gap-4">
-        <p class="text-text-secondary text-sm">
-          Still have an identity number?
-        </p>
-        <button
-          onclick={migrate}
-          class="text-text-primary text-sm font-semibold outline-0 hover:underline focus-visible:underline"
-        >
-          Upgrade
-        </button>
-      </div>
-    {:else}
-      <Button
-        onclick={migrate}
-        variant="tertiary"
-        size="xl"
-        disabled={!supportsPasskeys ||
-          nonNullish(authenticatingProviderId) ||
-          isGoogleAuthenticating}
-      >
-        Upgrade from legacy identity
-      </Button>
-    {/if}
+  </div>
+  <div class="border-border-tertiary border-t"></div>
```

Comment:
Moved out of the div with the buttons because it uses different spacing than between buttons.